by-language/csharp-npgsql/tests/DemoProgramTest.cs

```diff
@@ -79,6 +81,91 @@ public async Task TestUnnestExample()
             // Check results.
             Assert.Equal(10, resultCount);
         }
-        
+
+        [Fact]
+        public async Task TestAllTypesExample()
+        {
+            var conn = fixture.Db;
+
+            // Invoke database workload.
+            var task = DatabaseWorkloadsMore.AllTypesExample(conn);
+            var dt = await task.WaitAsync(TimeSpan.FromSeconds(0.5));
+
+            // Check results.
+            var row = dt.Rows[0];
+
+            // Numeric types
+            Assert.Equal(DBNull.Value, row["null_integer"]);
+            Assert.Equal(42, row["integer"]);
+            Assert.Equal((Int64) 42, row["bigint"]);
+            //Assert.Equal(42.42, (float) row["float"], 0.01);
```

Comment:
Why are these asserts disabled?

Reply:
It was probably just an oversight while erecting this a bit in haste. Thanks for spotting.

Reply:
Fixed per cd5c97c7f5d.